src/components/Markdown.tsx

```diff
@@ -0,0 +1,42 @@
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+import rehypeRaw from "rehype-raw";
+import rehypeSanitize, { Options } from "rehype-sanitize";
+import { useMemo } from "react";
+import { PluggableList } from "unified";
+
+export interface MarkdownProps {
+  /** Stringified Github-Flavored Markdown. */
+  content: string;
+  /** HTML Sanitization Schema for use with Rehype. */
+  sanitizationSchema?: Options;
+}
+
+/**
+ * Renders Github-Flavored Markdown from the Knowledge Graph. This Markdown can include
+ * arbitrary HTML. Any HTML will be sanitized according to the provided Rehype Schema. If
+ * no Schema is provided, Rehype's default will be used.
+ */
+export function Markdown({ content, sanitizationSchema }: MarkdownProps) {
+  const plugins: { remark: PluggableList; rehype: PluggableList } = {
```

Comment:
I don't think eslint will catch this, but this breaks the [react-perf/no-new-array-as-prop](https://github.com/cvazac/eslint-plugin-react-perf/blob/master/docs/rules/jsx-no-new-array-as-prop.md) rule since time this component is rendered new arrays will be created and passed as props. We can circumvent this by either moving the remark plugins out of the component body, or by wrapping everything in a memo like before.